### PR TITLE
Gutenboarding: Scaffold style preview

### PR DESCRIPTION
**Pull request by sirreal**
#### Changes proposed in this Pull Request

Basic layout of style preview.

Screen Shot 2020-03-13 at 17 49 54

This is very rough, but establishes components and a basic layout that makes more isolated and iterative work on the details much simpler.

#### Testing instructions

*This is behind a feature flag and only available in development.*

* [/gutenboarding](http://calypso.localhost:3000/gutenboarding)

Make sure production and development aren't broken. Otherwise this is not polished work and should not be reviewed in great detail.

Fixes https://github.com/Automattic/wp-calypso/issues/40009

### CODE REVIEW

File: client/landing/gutenboarding/onboarding-block/style-preview/viewport-select.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ */
+import * as React from 'react';
+import { SVG, Path, Rect, Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './viewport-select.scss';
+
+// TODO FIXME: React elements are a poor choice for static svgs.
```
Comment (simison) [inline client/landing/gutenboarding/onboarding-block/style-preview/viewport-select.tsx]: >  React elements are a poor choice for static svgs.

+1, can be in this PR or follow up.

File: client/landing/gutenboarding/components/titles.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../mixins';
+
+.gutenboarding-title,
+.gutenboarding-subtitle {
```
Comment (simison) [inline client/landing/gutenboarding/components/titles.scss]: Nice! 👌

File: client/landing/gutenboarding/components/titles.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	color: var( --mainColor );
```
Comment (blowery) [inline client/landing/gutenboarding/components/titles.scss]: who declares this one? seems like an overly generic name.

Comment (simison) [reply client/landing/gutenboarding/components/titles.scss]: Yup, we'll rename. 👍 Thanks :-)

https://github.com/Automattic/wp-calypso/issues/40085
https://github.com/Automattic/wp-calypso/pull/39972#discussion_r391119207

Comment (sirreal) [reply client/landing/gutenboarding/components/titles.scss]: It probably is! That's from https://github.com/Automattic/wp-calypso/pull/39928.

I'll add an issue to use a more specific variable for the color.

Comment (sirreal) [reply client/landing/gutenboarding/components/titles.scss]: For the record, the variables are here:

https://github.com/Automattic/wp-calypso/blob/cdf2cd28a91c272a268d2d6b774b9cbe9ca9dc56/client/landing/gutenboarding/onboarding-block/colors.scss

**Pull request merged**